app/helpers/application_helper.rb

```diff
@@ -2,4 +2,12 @@ module ApplicationHelper
   def max_width
     devise_controller? ? "mw-md" : "mw-xl"
   end
+
+  def color
```

Comment:
`bg_color` にしたほうがクラスの内容が分かりやすいでしょう。

```suggestion
  def bg_color
```